ANSIBLE.md

```diff
@@ -0,0 +1,27 @@
+#Steps to run the ansible script from local host on a remote server
+
+###Install ansible on your local machine.
+###Git clone this repository
+###Once the repository is cloned to your local machine, navigate to the ansible directory.
+###You will find a playbook.yml which is the base ansible script to run different roles(tasks) specified in it.
```

Comment:
nit, when referencing filenames (or proper noun in general), use the backtick character (shift + ~) before and after the filename eg `playbook.yml`